<?xml version="1.0"?>
<launch>
    <arg name="topic" default="/random/float"/>
    <arg name="min_val" default="1"/>
    <arg name="max_val" default="15"/>
    <arg name="rate" default="1"/>

    <node name="float_publisher" pkg="roshell_graphics" type="float_publisher_node" output="screen">
        <param name="topic" value="$(arg topic)"/>
        <param name="min_val" value="$(arg min_val)"/>
        <param name="max_val" value="$(arg max_val)"/>
        <param name="rate" value="$(arg rate)"/>
    </node>

</launch>